src/pybind11-qt/pybind11_qt_basic.cpp

```diff
@@ -46,6 +46,10 @@ namespace pybind11::detail {
         PyObject* strPtr =
             PyUnicode_Check(objPtr) ? PyUnicode_AsUTF8String(objPtr) : objPtr;
 
+        if (!strPtr) {
+            return false;
```

Comment:
I guess this is the fix for the crashes but can you tell what was in `objPtr` when this occurs?

Reply:
It's a string object, but the string format returned when there are emojis fails to properly convert back into a QString. The above change prevents you from having to convert the string before passing it back to C++.

This is described in more detail in #plugins-dev.